src/content/en/tools/lighthouse/audits/title.md

```diff
@@ -0,0 +1,74 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Document Doesn't Have A Title Element" Lighthouse audit.
+
+{# wf_updated_on: 2018-03-02 #}
+{# wf_published_on: 2018-03-02 #}
+{# wf_blink_components: Platform>DevTools #}
+
+# Document Doesn't Have A Title Element  {: .page-title }
+
+## Overview {: #overview }
+
+Note: This audit occurs in both the Accessibility and SEO categories of your
+Lighthouse report.
+
+### Accessibility {: #accessibility }
+
+The document title gives screen readers users an overview of a page's purpose.
```

Comment:
`screen readers users` => `users of screen readers`?

Reply:
I guess it gives _everyone_ an overview of the page's purpose, but maybe say that it's especially useful to screen reader users who have limited context about the page.